app/playbooks/deploy.d/odh-demo/odh-demo.yml

```diff
@@ -1,8 +1,8 @@
 ---
```

Comment:
probably best to also remove the odh-demo. It is the old spam mail demo and probably does not work anymore

rm -rf app/playbooks/deploy.d/odh-demo